python/lsst/pipe/base/pipelineIR.py

```diff
@@ -837,7 +837,7 @@ def from_string(cls, pipeline_string: str):
         reason="This has been replaced with `from_uri`. will be removed after v23",
         version="v21.0,",
         category=FutureWarning,
-    )  # type: ignore
+    )
```

Comment:
In theory we can now remove this since v23 is out...

Reply:
I'll save this for another ticket I already have open (the one on expanded pipelines).